javascript/javascript2/week3/js/fastestPresser.js

```diff
@@ -0,0 +1,100 @@
+const input = document.getElementById('seconds')
+const start = document.getElementById('start')
+const scoreS = document.querySelector('.player-S p:nth-child(2)')
+const scoreL = document.querySelector('.player-L p:nth-child(2)')
+const result = document.getElementById('result')
+const timer = document.getElementById('timer')
+const reset = document.getElementById('new-game')
+const confettiS = document.getElementById('canvas-S');
+const confettiL = document.getElementById('canvas-L');
+
+let countS = 0
+let countL = 0 
+
+const startGame = () => {
+    const input = +document.getElementById('seconds').value
+    newGame()
+    eventListener()
+    countDown(input)
+    gameResult(input)
+}
+
+const newGame = () => {
+    countS = 0
+    countL = 0
+    scoreS.innerText = ''
+    scoreL.innerText = ''
+    timer.innerText = ''
+    result.innerText = ''
+}
+
+const resetGame = () => {
+    input.value = ''
+    newGame()
+}
+
+const eventListener = () => {
+    document.addEventListener("keydown", scoreCounter)
+}
+
+const scoreCounter = (event) => {
+    if (event.key === "s") {
+        countS++
+        scoreS.innerText = countS
+    } else if (event.key === "l") {
+        countL++
+        scoreL.innerText = countL
+    }
+}
+
+const countDown = (seconds) => {
+    let timeLeft = seconds
+    const timeCount = setInterval(() => {
+        timeLeft--
+        timer.innerText = `${timeLeft} seconds left`
+        if (timeLeft === 0) {
+            clearInterval(timeCount)
+            timer.innerText = 'time up'
+        }
+        if (timeLeft === -1) {
+            clearInterval(timeCount)
+            timer.innerText = ''
+        }
```

Comment:
I don't think this is required, when time =0 if will already do the clean up

Reply:
I agree with it. I will recheck and remove the clearInterval.

Thank you Diny.